cdap-docs/application-templates/source/etl/templates/sinks/real-time/cube.rst

```diff
@@ -16,3 +18,34 @@ sink.
 
 To configure transformation from a StructuredRecord to a CubeFact, the mapping
 configuration is required, following StructuredRecordToCubeFact documentation.
+
+.. rubric:: Use Case
+
+TODO: Fill me out
+
+.. rubric:: Properties
+
+**name:** Name of the Cube dataset. If the Cube does not already exist, one will be created.
+
+**dataset.cube.resolutions:** Aggregation resolutions to be used if new Cube dataset needs to be created.
+See `Cube dataset configuration details <http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/cube.html#cube-configuration>`__ for more information.
+
+**dataset.cube.properties:** Provide any dataset properties to be used if new Cube dataset
+needs to be created as a JSON Map. For example if aggregations are desired on fields - abc and xyz, the
+property should have the value: {"dataset.cube.aggregation.agg1.dimensions":"abc", "dataset.cube.aggregation.agg2.dimensions":"xyz"}.
+See `Cube dataset configuration details <http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/cube.html#cube-configuration>`__ for more information.
+
+**cubeFact.timestamp.field:** Name of the StructuredRecord's field that contains timestamp to be used in CubeFact.
```

Comment:
Name of the StructuredRecord field that contains the timestamp to be used in the CubeFact.
If not provided, the current time of the record processing will be used as the CubeFact timestamp.